config/example_config.yml

```diff
@@ -45,17 +45,28 @@ Data:
         #   - 'SSTA': SST-2M ASTRI Telescope and Camera
         #   - 'SSTC': SST-2M GCT Telescope with CHEC Camera
         #   - 'VTS': VERITAS Telescope and Camera
-        # Telescope type to load.
-        selected_tel_type: 'LST'
+        # List of telescope types to load.
+        selected_tel_types: ['LST']
+        
+        # Optional dictionary with string keys and list of integer values,
+        # or null. If null, default is an empty dictionary.
+        # Valid keys are the tel_types listed above, and the values should be
+        # the tel_ids of the telescopes to load of that tel_type.
+        # Default if tel_type not in dict: load all telescopes of the tel_type.
```

Comment:
Maybe the two options selected_tel_types and selected_tel_ids could be combined somehow. That way there wouldn't be the redundancy of having tel types in selected_tel_types as well as keys in selected_tel_ids. For example, one field called 'selected_tels', which is a dict just like selected_tel_ids. Including a telescope type as a key indicates that the tel type is selected. If the value is null, then all telescopes are selected. If the value is a list of IDs, only those are included.

Reply:
I considered this approach, but decided against it for the following reasons. First, if you don't care about selecting tel_ids (probably the case for most observatories like VERITAS with real data, I think this feature is most useful for selecting one subarray from a simulation containing multiple subarrays), it's annoying and confusing to have to write null after the tel type. 

Second, since I anticipate that the selected tel ids would usually correspond to well-defined subarrays, they probably would have just a couple of standard values that wouldn't be changed unless you're specifically studying this parameter. It would be much more common to just change which tel types are selected. In that case, it's more convenient to just have to comment out a single tel type than a whole dict of selected tel ids. This is similar to how Image Mapping:padding is treated now.